core/data-plane/data-plane-common/README.md

```diff
@@ -0,0 +1,3 @@
+# Data Plane Common
+
+Utility module for the data-plane
```

Comment:
If it's a utility module, we should call it `dataplane-util`, otherwise I would describe it as "module that contains common classes and functionality", or something similar.